platforms/hyperledger-besu/configuration/deploy-network.yaml

```diff
@@ -44,10 +44,14 @@
       git_dir: "{{ organizationItem.gitops.release_dir }}"
       org: "{{ organizationItem }}"
       kubernetes: "{{ organizationItem.k8s }}"
+      cloudProvider: "{{ organizationItem.cloud_provider }}"
+      cloudProvider_info: "{{ organizationItem[cloudProvider] }}"
+      charts_dir: "platforms/shared/charts"
```

Comment:
can this be relative path from {{ playbook_dir }}

Reply:
I checked on this however the playbook_directory variable doesn't support directly accessing the helm chart path from our current working directory (/home/<user-name>/<bevel-directory-name>). If we use playbook_dir, it will start from the /home directory, which is not ideal.